frontend/ui/widget/bookmapwidget.lua

```diff
@@ -1341,13 +1346,16 @@ function BookMapWidget:showMenu()
     }
     UIManager:show(button_dialog)
 end
+
+function BookMapWidget:onShowMenu()
+    self:showMenu()
```

Comment:
Maybe just rename `showMenu()` to `onShowMenu()`?

Reply:
Bad advise (and bad code). We have
https://github.com/koreader/koreader/blob/2f0e456a428ef81de7c0747208d581f39e9359da/frontend/dispatcher.lua#L63

Reply:
Renaming `showMenu()` to `onShowBookMapMenu()` looks good.